src/pages/workflows/workflow-details/WorkflowWrapper.ts

```diff
@@ -291,7 +291,7 @@ export const WorkflowsContainer = (props: WorkflowContainerProps) => {
       }),
     permissionsModalOpen &&
       h(PermissionsModal, {
-        versionOrNamespace: 'Version',
+        versionOrCollection: 'Version',
```

Comment:
Incredibly glad we avoided working on snapshot => version and namespace => collection at the exact same time.